### DISCUSSION TITLE
Error on Logout - "invalid parameter: redirect_uri"

### DISCUSSION THREAD

**Discussion by amitmahajan82** [keycloak/keycloak#17269] [Q&A] [answered]
Hi All,

I am new to Keycloak and trying with to run Keycloak 20.0.3 in dev mode (start_dev) using official docker image . I am using the following URL to logout a user:
_http://127.0.0.1:8080/realms/<realm-name>/protocol/openid-connect/logout?redirect_uri=http://127.0.0.1:8081_

But Keycloak is giving the following error:
_invalid parameter: redirect_uri_

Am I doing something wrong?

Thanks,
Amit

**Comment by sschu** [comment, marked as answer]
The correct parameter for logout is `post_logout_redirect_uri`, see https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout

**Reply by amitmahajan82** [reply]
Thanks for pointing out the correct parameter. It worked :)
